fusor-ember-cli/app/controllers/register-nodes.js

```diff
@@ -440,12 +480,11 @@ export default Ember.Controller.extend({
         "X-CSRF-Token": token,
       },
       data: JSON.stringify({ 'node': createdNode })
-    }).then(function(registeredNode) {
+    }).then(function(result) {
         // node was added on the backend, but model.nodes needs to be freshed
         self.send('refreshNodesAndFlavors');
         self.set('initRegInProcess', false);
-        self.addIntrospectionNode(registeredNode);
-        self.doNextNodeRegistration(registeredNode);
```

Comment:
@sdherr, I assume these two lines are to be deleted since there is no `registeredNode` object that is returned